xchange-core/src/main/java/org/knowm/xchange/dto/Order.java

```diff
@@ -185,6 +197,7 @@ public Date getTimestamp() {
     return timestamp;
   }
 
+  @JsonProperty("flags")
```

Comment:
property names in JSON should be the same as in the classes

Reply:
OK, for consistency I have renamed the field to match the property :)

Reply:
I've also had to add a `JsonProperty("orderFlags")` on the builder.

Basically I'm aware of the convention but:

- Builder was using `flags`
- Field was called `flags`
- Getter and setter were called `getOrderFlags/setOrderFlags`

So I went with the most common one!

I've reversed this to standardise them all on `orderFlags`.